README.md

```diff
@@ -14,15 +14,25 @@ You need the ability to run docker and for it to be active in your session to ru
 ```
```

Comment:
After the first paragraph add

__Note:__ These instructions are only applicable for OSX and may be different for your system.